src/current/cockroachcloud/basic-cluster-management.md

```diff
@@ -98,17 +77,17 @@ To set the primary region:
 
 ### Move cluster to a new region
 
-A CockroachDB {{ site.data.products.serverless }} cluster cannot be moved from one region to another directly. Instead, you must restore the cluster's data from a backup to a new CockroachDB {{ site.data.products.serverless }} cluster with the desired region configuration:
+A CockroachDB {{ site.data.products.basic }} cluster cannot be moved from one region to another directly. Instead, you must restore the cluster's data from a backup to a new CockroachDB {{ site.data.products.basic }} cluster with the desired region configuration: 
```

Comment:
Nit: Trailing whitespace. You may be able to configure your editor to automatically trim it.